app/repository/config.py

```diff
@@ -89,24 +90,45 @@ def _to_corpus_data(row, event_types) -> CorpusData:
     )
 
 
-def get_corpora_for_org(db: Session, org_id: int) -> Sequence[CorpusData]:
-    corpora = (
-        db.query(
-            Corpus.import_id.label("corpus_import_id"),
-            Corpus.title.label("title"),
-            Corpus.description.label("description"),
-            Corpus.corpus_type_name.label("corpus_type"),
-            CorpusType.description.label("corpus_type_description"),
-            CorpusType.valid_metadata.label("taxonomy"),
+def get_corpora(
+    db: Session, user_email: str, is_superuser: bool
+) -> Sequence[CorpusData]:
+    if is_superuser:
```

Comment:
DRY: can we not pull out the main common part of the query from the if/else clause?

Reply:
Done